webpack.config.js

```diff
@@ -2,13 +2,16 @@ const path = require('path');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 
-const genConfig = (filename, { inlineCss } = {}) => ({
+const genConfig = (outputName, { inlineCss, inputName = 'index.js' } = {}) => ({
   output: {
     path: path.resolve(__dirname, 'dist'),
-    filename,
+    filename: outputName,
     library: 'superheroUtils',
     libraryTarget: 'umd'
   },
+  entry: {
+    [inputName.slice(0, -3)]: `./src/${inputName}`
+  },
```

Comment:
```suggestion
  entry: `./src/${inputName}`,
```
https://webpack.js.org/concepts/entry-points/#single-entry-shorthand-syntax

Reply:
Why don't do exactly as I proposed?

Reply:
Not sure what do you mean

I changed it to the 
```js
    entry: `./src/${inputName}`,
```
as I understand it's exactly as you proposed